src/VecSim/types/bfloat16.h

```diff
@@ -13,7 +13,7 @@
 namespace vecsim_types {
 struct bfloat16 {
     uint16_t val;
-    constexpr bfloat16() = default;
+    bfloat16() = default;
```

Comment:
Is this a compiler version issue? How come it works with fp16? Can we try this option?
```suggestion
    constexpr bfloat16() noexcept {}
```

Reply:
declaring a default ctor as constexpr is allowed as of c++20, and this feature in particular is supported as of gcc10
fp16 worked because it didn't have constexpr at first place.
Anyway, since we have another ctor that is constexpr, it doesn't matter.